packages/mui-material/src/Popover/Popover.test.js

```diff
@@ -902,4 +962,136 @@ describe('<Popover />', () => {
       ),
     ).not.to.throw();
   });
+
+  describe('prop: slots', () => {
```

Comment:
@DiegoAndai I think testing `slots` and `slotProps` can be covered by `describeConformance` [here](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Popover/Popover.test.js#L41-L57), you just need to adjust the config ~

Reply:
Thanks @mj12albert! didn't know about that 😅

Reply:
👌 Just learned about it [in my own PR](https://github.com/mui/material-ui/pull/36119#discussion_r1199247387) as well, I haven't fully explored the internals of the conformance tests myself!